src/provider/zeebe/properties/FormProps.js

```diff
@@ -96,8 +96,8 @@ function FormType(props) {
   const getOptions = () => {
     return [
       { value: '', label: translate('<none>') },
-      { value: FORM_TYPES.CAMUNDA_FORM_LINKED, label: translate('Camunda form (linked)') },
-      { value: FORM_TYPES.CAMUNDA_FORM_EMBEDDED, label: translate('Camunda form (embedded)') },
+      { value: FORM_TYPES.CAMUNDA_FORM_LINKED, label: translate('Camunda Forms (linked)') },
```

Comment:
Let's keep this singular, i.e. call it `Camunda Form` instead of `Camunda Forms`. We're not linking multiple here.